src/Program.cs

```diff
@@ -182,7 +183,7 @@ private static async Task<int> Main(string[] args)
                     formatOptions,
                     logger,
                     cancellationTokenSource.Token,
-                    createBinaryLog: logLevel == LogLevel.Trace).ConfigureAwait(false);
+                    binaryLogPath: binarylog).ConfigureAwait(false);
```

Comment:
If including the default value causes it to always provide a binarylog path, you could check for the option specifically.

```suggestion
                    binaryLogPath: s_parseResult.WasOptionUsed("--binarylog") ? binarylog : null).ConfigureAwait(false);
```

Reply:
It may be better to not provide a default value for the option and to set a default if `binarylog is null` and the `--binarylog` option was used...